protocols/medco/deterministic_switching_test.go

```diff
@@ -13,7 +13,6 @@ import (
 )
 
 func TestDeterministicSwitching(t *testing.T) {
-	t.Skip("Skipping due to error in medco: https://github.com/dedis/cothority/issues/479")
```

Comment:
I'll do extensive tests (on travis). If they all pass repeatedly we even close #479